fv3gfs-physics/fv3gfs/physics/stencils/fv_update_phys.py

```diff
@@ -146,8 +151,20 @@ def __call__(
             self._dt,
         )
         # [TODO] needs a better solution to handle u_dt, v_dt quantity
-        u_dt_quantity = self.grid.make_quantity(u_dt)
-        v_dt_quantity = self.grid.make_quantity(v_dt)
+        u_dt_quantity = fv3gfs.util.Quantity(
```

Comment:
Still a TODO for later but yeah, we should really not have to make a quantity to turn into a storage and then back into a quantity.

Reply:
This should go away when we have proper PhysicsState, and each variable has a quantity and storage.